include/WCSimWCTrigger.hh

```diff
@@ -83,6 +83,8 @@ public:
   ///Knowledge of the dark rate (use for automatically adjusting for noise)
   void SetDarkRate(double idarkrate){ PMTDarkRate = idarkrate; }
 
+  int event_number;
```

Comment:
Could you add comments to these in doxygen format? Specifying that event_number starts at "1" will also be helpful.